packages/remix/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@logto/remix",
-  "version": "2.1.9",
+  "version": "2.1.10",
```

Comment:
According to [semver](https://semver.org/#summary), this change should be a MINOR version bump instead of PATCH version bump. Should be `2.2.0` IMO.

Reply:
Thanks, agree and fixed

Reply:
oops, too quick, i need to change it to 2.2.0 instead of 2.2.9...